docs/source/plugins/01_quick_start.rst

```diff
@@ -104,22 +96,37 @@ The highlights here are for:
 :myplugin.py: Implementation of the hooks for customization of the UI interface, or the solver configuration hooks.
 :myplugin.c:  Implementation of the hooks for customization of solver.
 
-
 Check out the :ref:`Plugin Structure section <plugin_structure-section>` for more details about how the folder and files are structured, and
 also, check the :ref:`plugin-by-example-section` that shows how to create simple plugins that make use of the `User Interface Hooks` and the `Solver Hooks`.
 
+Invoke support
+--------------
+
+From version 0.17.0 onwards, |sdk| have ``pyinvoke`` support in order facilitate and standardize the plugins development process
+using pre-defined tasks. In order to use it, you should create a new plugin with the command below and a ``tasks.py`` file will be generated.
+
+The generetad ``tasks.py`` file can be used to implement your own tasks and to overwrite the default tasks defined in the ``alfasim_sdk.default_tasks`` .
+
+|sdk| already have some tasks available out of the box that will help to speed up your workflow. You might list them using:
+
+.. code-block:: bash
+
+    >>> cd myplugin
+    >>> invoke --list
+
+Further details about ``pyinvoke`` can be found on its `documentation <https://www.pyinvoke.org/>`_.
```

Comment:
This will make the `pyinvoke_` link I used in the previous suggestion work:

```suggestion
.. _pyinvoke: https://www.pyinvoke.org
```

Reply:
Nice.